### PR TITLE
Banner

### PR DESCRIPTION
**Pull request by PeteW**
# PR Prelude

Thank you for working on SpaceVim! :)

**Please complete these steps and check these boxes (by putting an `x` inside
the brackets) _before_ filing your PR:**

- [x] I have read and understood SpaceVim's [CONTRIBUTING][cont] document.
- [x] I have read and understood SpaceVim's [CODE_OF_CONDUCT][code] document.
- [x] I have included tests for the changes in my PR. If not, I have included a
  rationale for why I haven't.
- [x] **I understand my PR may be closed if it becomes obvious I didn't
  actually perform all of these steps.**

# Why this change is necessary and useful
banners are cool!

### CODE REVIEW

File: autoload/SpaceVim/layers/core/banner.vim [hunk 1]
```diff
@@ -390,6 +390,44 @@ function! SpaceVim#layers#core#banner#config() abort
...
+        \ 'VIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMWKo.;OKK0KKKKKKK0x;................:kX',
+        \ 'VIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMMMWl:OKKKKK0KK0k:..:ll;..col.coo:;lxoo',
+        \ 'VIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMMMWo:OKKKKKK0kc;..l00o..:Ox:.d0kdodKxo',
+        \ 'VIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMMMWo:OKKKK0kc;...:OXx;.,xk; .xkoxodOlk',
+        \ 'VIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMMMWo:0KK0kl;....,d0Oc..lOc..:Ol:llko:O',
+        \ 'VIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMMMMd,lolc,cdo,..........cdkkxkxxxdkxxX',
+        \ 'VIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMMMMNkooooONMWKo,......;xXMMMMMMMMMMMMM',
+        \ 'VIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMMMMMMMMMMMMMMMWKo,..;dXMMMMMMMMMMMMMMM',
+        \ 'VIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMMMMMMMMMMMMMMMMMWO.dKMMMMMMMMMMMMMMMMM',
```
Comment (wsdjeg) [inline autoload/SpaceVim/layers/core/banner.vim]: I think maybe it is better to move logo to the center, if the screen is too narrow, we can not see the logo.

**Review by wsdjeg — COMMENTED** [review]
Thanks for your PR.

File: autoload/SpaceVim/layers/core/banner.vim (hunk 1 above)
Comment (PeteW) [reply autoload/SpaceVim/layers/core/banner.vim]: @wsdjeg agreed thanks. here is another attempt with more respect to smaller resolutions

**Comment by wsdjeg** [comment]
Thanks. merged.

**Comment by PeteW** [comment]
thanks! banner work is good practice to making more meaningful contributions in the future :)

**Comment by wsdjeg** [comment]
@PeteW Yeah, both programmer and artist.

**Pull request merged**